piaplib/pku/__init__.py

```diff
@@ -29,20 +29,14 @@
 	del ImportErr
 	raise ImportError(u'Pocket Knife Unit PKU failed to import.')
 
+
 try:
 	from . import pku as pku
-	if pku.__name__ is None:
+	if pku.__name__ is False:
```

Comment:
This was a drive-by-fix, is it really necessary? This should have a test case built for it.